jjb/edgex-templates-pipeline.yaml

```diff
@@ -67,19 +65,20 @@
       - github-pull-request:
           trigger-phrase: '^recheck$'
           only-trigger-phrase: false
-          status-context: '{status-context}'
+          status-context: '{project-name}-verify-pipeline'
           permit-all: false
           github-hooks: true
           org-list:
             - '{github-org}'
+          allow-whitelist-orgs-as-admins: true
           auto-close-on-fail: false
           white-list-target-branches:
             - '{branch}'
 
 
 - pipeline_merge_boiler_plate: &pipeline_merge_boiler_plate
     name: pipeline_merge_boiler_plate
-
+    # concurrent: true
```

Comment:
this was helpful in side-by-side comparison in split view.

also, @lranjbar, should we consider turning this on?

Reply:
Should be fine to turn this back on. When I turned this off the template still was building from two Jenkinsfiles for verify and merge and this was to guard against a potential race condition.

Reply:
in my update, i nixed this line from the merge boilerplate as well as from the verify boilerplate because it is set to true in the common job boilerplate